platforms/Linux/connection.c

```diff
@@ -36,7 +47,8 @@ int create_socket(const char * portStr, int addressFamily)
     hints.ai_socktype = SOCK_DGRAM;
     hints.ai_flags = AI_PASSIVE;
 
-    if (0 != getaddrinfo(NULL, portStr, &hints, &res))
+	int r = getaddrinfo(NULL, portStr, &hints, &res);
+    if (0 != r)
```

Comment:
No not at all. Had to do that while debugging to get the error code. I'll remove it.